nasa-gibs/worldview
# GITC WV 4.11.0 Build

## Description

Updates to the build process and some UI updates to be able to run the latest version of Worldview through GITC.

## How To Test

1. `git checkout GITC-WV-4.11.0`
2. `npm ci`
3. `npm run build`
4. Load WV locally and ensure that it has built correctly..
